bundle/manifests/kuadrant.io_dnspolicies.yaml

```diff
@@ -69,37 +69,17 @@ spec:
               healthCheck:
                 description: |-
                   HealthCheckSpec configures health checks in the DNS provider.
-                  By default, this health check will be applied to each unique DNS A Record for
+                  By default this health check will be applied to each unique DNS A Record for
                   the listeners assigned to the target gateway
                 properties:
                   endpoint:
-                    description: |-
```

Comment:
it doesn't seem great to have lost the field descriptions, can they be re-added to the Spec in the DNS Operator to keep them?

Reply:
added those to the DNS Operator bu we will see them here only after the dns-operator PR gets merged

Reply:
@philbrookes not should be fine